.gitignore

```diff
@@ -3,4 +3,5 @@ coverage
 test.js
 test.ts
 test-manager.js
-dist/
\ No newline at end of file
+dist/
+.idea
```

Comment:
Since this folder doesn't belong to the project, please remove the reference from .gitignore and don't upload the folder with your PR.